doc/py3status.html

```diff
@@ -8,143 +8,198 @@ <h1>User-contributed article: enhanced and extensible i3bar with py3status</h1>
+    Therefore, there is no module to run arbitrary scripts or commands.
+    Instead, you should use your shell.
+</tt></pre>
 
 <h2>Introducing py3status</h2>
 
 <p>
 The goal of py3status is to fill this gap by allowing users to simply extend
-their i3bar while preserving their current i3status configuration. The main idea
-is to rely on i3status' strength without adding any configuration on the user's
-side. py3status is thus a wrapper script for i3status and its configuration as
-explained <a href="https://github.com/ultrabug/py3status/tree/master/doc">
-in the documentation</a>.
+their i3bar <b>while preserving their current i3status configuration</b>.
+The main idea is to rely on i3status' strength without adding any configuration
+on the user's side. py3status is thus a wrapper script for i3status and its
+configuration as explained in the documentation below.
+This should be said that py3status is much more than just a wrapper script.
 </p>
 
-<h2>Usage</h2>
+<h2>Documentation</h2>
 
-<p>
-Using py3status is easy, no need to multi-pipe your scripts after i3status.
-Instead just replace <i>i3status</i> in your current <b>status_command</b> by
- <i>py3status</i>.
-For example, if your current status_command in your i3 config file resides in
-~/.i3/i3status.conf, you would change your i3 config to this:
-</p>
+<p>View the full documentation now or bookmark it for later viewing:</p>
+<ul>
+<li>
+    <a href="https://py3status.readthedocs.io/en/latest">Read the Docs</a>
+</li>
+<li>
+    <a href="https://github.com/ultrabug/py3status/tree/master/doc">Github</a>
+</li>
+</ul>
 
-<pre><tt>status_command py3status -c ~/.i3/i3status.conf</tt></pre>
+<h2>Using py3status</h2>
 
-<h2>Handle i3bar click events from your i3status.conf</h2>
+<p>Using py3status has never been so easy. Update your i3 config with this and
+you're done:</p>
+<pre><tt>
+bar {
+    status_command py3status
+}
+</tt></pre>
 
 <p>
-Py3status (since v2) is also wrapping and extending your i3status.conf and
-allows you directly handle all the i3bar click events on any of your configured
-modules whether they are i3status modules or py3status modules.
+In other words, just replace <i>i3status</i> in your current
+<b>status_command</b> by <i>py3status</i>.
 </p>
 
-<p>
-To do so, all you have to do is add a new configuration parameter named
-<b>on_click [button number]</b> to your module config and py3status will then
-execute the given i3 command (using i3-msg). This means you can run simple
-tasks like executing a program or execute any other i3 specific command.
-</p>
-<p>Some examples below from i3status.conf:</p>
+<p>Py3status will look for status config file in this order.</p>
+<pre><tt>
+1    $HOME/.i3/i3status.conf
+2    $HOME/.i3status.conf
+3    /etc/i3status.conf
+4    $XDG_CONFIG_HOME/.config/i3status/config
+5    $HOME/.config/i3status/config
+6    $XDG_CONFIG_DIRS/i3status/config
+7    /etc/xdg/i3status/config
```

Comment:
Maybe worth making a decision on https://github.com/ultrabug/py3status/pull/1694/ first, and if convinced updating this piece and merging both at the same time... 😉

Reply:
I want us to stay in line with i3status code base and config patch detection order.

I do not remember what it is today, I would need to read i3status code again.

But zen says : first i3status compliant, then add our stuff

Reply:
I think the order should be the exact i3status configs order _prepended in front_ with a single py3status config file. This way if py3status-specific config exists, it is used, otherwise we fallback to i3status. Sway works the same way, by the way, it is compliant with i3 config, but if you create sway-specific config file, it is used instead. The reasoning is exactly the same as in py3status, while sway/py3status _can_ work with i3/i3status configs, the opposite is not true.

By the way, if we support `~/.config/py3status/py3status.conf`, the section

```
<p>Alternatively, you can start from scratch with an empty status config.</p>
status_command py3status -c ~/.config/i3/py3status.conf
```

Will have an alternative approach:

```
<p>Alternatively, you can start from scratch with an empty status config.</p>
touch ~/.config/py3status/py3status.conf (and keep plain "status_command py3status")
```

Reply:
In #1694, I mentioned i3status and i3 made mistakes with the list. We could not list them in an explicit order... just that it would look for a configuration in a similar fashion as i3status.

Reply:
done